package.json

```diff
@@ -13,16 +13,16 @@
       "default": "./dist/index.mjs"
     },
     "./rules-by-category": {
-      "types": "./dist/rules-by-category.d.ts",
-      "import": "./dist/rules-by-category.mjs",
-      "require": "./dist/rules-by-category.cjs",
-      "default": "./dist/rules-by-category.mjs"
+      "types": "./dist/generated/rules-by-category.d.ts",
```

Comment:
My only worried line. I tested it with `npm link` but another pair of eyes is always better ^^